contrib/gitlab/src/mill/contrib/gitlab/GitlabHttpApi.scala

```diff
@@ -0,0 +1,32 @@
+package mill.contrib.gitlab
+
+import scala.concurrent.duration._
+
+object GitlabUploader {
+  type Upload = (String, Array[Byte]) => requests.Response
```

Comment:
It would be nice to explicitly add the `requests` dependency to the modules `ivyDeps`. It's currently only transitively available, which may change and break thinks.

Reply:
Will add 👍 . This depends from from `os` also. Should it be added also?

Reply:
Added all. Added to artifactory, bintray and codearticat also since they depend from requests also.